pycaret/classification.py

```diff
@@ -9638,6 +9687,32 @@ def deploy_model(model,
     - Default Region Name (can be seen under Global settings on your AWS console)
     - Default output format (must be left blank)
 
+    For GCP users:
+    --------------
```

Comment:
I am pretty sure you cannot use headers like that inside doc sections - I'd just remove it (`--------`)